environment.yml

```diff
@@ -7,12 +7,19 @@ channels:
   - defaults
 
 dependencies:
+  # conda-forge packages
+  - r=3.5.1
+  - r-rcolorbrewer=1.1
+  # bioconda packages
   - bcftools=1.8
+  - bioconductor-rtracklayer=1.42.1
   - bwa=0.7.17
+  - cancerit-allelecount=2.1.2
```

Comment:
Need to update Bioconda recipe.
We should at least have the 2.2.0

Reply:
https://github.com/bioconda/bioconda-recipes/tree/master/recipes/cancerit-allelecount